cardano-config/src/Cardano/Config/Partial.hs

```diff
@@ -211,15 +211,13 @@ data PartialWallet = PartialWallet
 checkComplete :: String -> Last a -> Either ConfigError a
 checkComplete name (Last x) = maybe (Left $ PartialConfigValue name) Right x
 
---
--- The finalise* family of functions are supposed to be called at the very last stage
--- in the partial options monoid approach, after all the parametrisation layers have been merged,
--- and we're intending to use the resultant config -- they ensure that all values are defined.
---
--- NOTE: we should look into applying generic programming and/or TH for this boilerlate.
---
-finaliseCardanoConfiguration :: PartialCardanoConfiguration -> Either ConfigError CardanoConfiguration
-finaliseCardanoConfiguration PartialCardanoConfiguration{..} = do
+-- This utilizes the check* family of functions to make sure we have all
```

Comment:
By the way, @Jimbo4350, the rest of functions are also still called `check`.. which, once again, doesn't quite fit their bill.

Reply:
I'll change them to `mk`*